framework/src/modules/chain/submodules/transport.js

```diff
@@ -698,7 +693,7 @@ Transport.prototype.shared = {
 				if (err) {
 					return setImmediate(cb, null, {
 						success: false,
-						message: err.message || 'Invalid transaction body',
+						message: err.message || 'Transaction was rejected with errors',
```

Comment:
if i understand correctly, @4miners found that the `err.message` was undefined as `err` was `type` `string`.
so can you handle the issue?

Reply:
@ManuGowda that assumption was not correct; err is sometimes an error and other times an array. so that part is correct.

Reply:
hmmm ok.. so that case as well.. `err.message` is wrong rite.. if its an array then `err.message` will result in undefined know? am i missing something?